### DISCUSSION TITLE
如何设置xmake project 生成的解决方案名字？

### DISCUSSION THREAD

**Discussion by Unreal-3D** [xmake-io/xmake#4915] [General]
生成出来的xcodeproj名字跟什么配置项相关呢，set_project无效

**Comment by waruqi** [comment]
目前不能配置，因为现在 xcode proj 其实是走 cmakelists 生成后，借助 cmake 转的，并不是直接生成的 xcodeproj。。没法细粒度控制。。

要等 https://github.com/xmake-io/xmake/issues/4810
